src/lib/mdns/platform/Mdns.h

```diff
@@ -144,6 +144,18 @@ CHIP_ERROR ChipMdnsPublishService(const MdnsService * service);
  */
 CHIP_ERROR ChipMdnsStopPublish();
 
+/**
+ * This function removes a service published by mDNS.
+ *
+ * @param[in] service   The service entry.
+ *
+ * @retval CHIP_NO_ERROR                The remove succeeds.
+ * @retval CHIP_ERROR_INVALID_ARGUMENT  The service is nullptr.
+ * @retval Error code                   The remove fails.
+ *
+ */
+CHIP_ERROR ChipMdnsRemoveService(const MdnsService * service);
```

Comment:
Should it be called `ChipMdnsStopPublishService` to match with `ChipMdnsStopPublish` ?

Reply:
Personally I don't like this naming with `StopPublish`, as for me it associates with stopping an action e.g. for some time and not removing some object, but anyway you are right. For now I will align with existing code, as I don't want to blindly rename single methods without having wider context.